docs/cep/sink/query-worker-sink.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 60
+title: Query Worker Sink
+---
+
+You can use a new or existing [query worker](../../queryworkers/query-workers) as a stream worker sink.
+
+For more information about writing queries and creating query workers, refer to [Queries and Query Workers](../../queryworkers/).
+
+## Syntax
+
+```sql
+CREATE SINK <NAME> WITH (type="query-worker", map.type="<STRING>", query.worker.name="<STRING>", sink.id="<STRING>")
+```
+
+## Query Parameters
+
+| Name     | Description                   | Default Value            | Possible Data Types | Optional | Dynamic |
+|----------|---------------------------|--------------------------|---------------------|----------|---------|
+| query.worker.name  | TThe name of an existing query worker. | | string | No | Yes |
```

Comment:
```suggestion
| query.worker.name  | The name of an existing query worker. | | string | No | Yes |
```

Reply:
Is one column supposed to be empty here?

Reply:
Yes, because there is no default value for either.